### PR TITLE
Therapeutics improvements from a readthrough.

### PR DESCRIPTION
**Pull request by cbrueffer**
### Description of the proposed additions or changes

Edits from a therapeutics readthrough; from a quick glance smaller and mostly complementary to Anthony's and Simina's PRs.

### Related issues

https://github.com/greenelab/covid19-review/pull/996
https://github.com/greenelab/covid19-review/pull/993

### Suggested reviewers (optional)

@rando2 @agitter @SiminaB @cgreene 

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by agitter** [comment]
The build failed because of the special character in the dictionary
```
Error: /home/appveyor/projects/covid19-review/build/assets/custom-dictionary.txt: The word "IFN-?-1a" is invalid. The character '-' (U+2D) may not appear in the middle of a word.
```

I may be able to troubleshoot that later today.

**Comment by rando2** [comment]
> The build failed because of the special character in the dictionary
> 
> ```
> Error: /home/appveyor/projects/covid19-review/build/assets/custom-dictionary.txt: The word "IFN-?-1a" is invalid. The character '-' (U+2D) may not appear in the middle of a word.
> ```
> 
> I may be able to troubleshoot that later today.

Is it the beta or the hyphen? [My read of the error message is that it doesn't like compound words!](https://www.fileformat.info/info/unicode/char/2d/index.htm) Perhaps we could just try removing it from the dictionary and seeing if it gets picked up by spell check.

**Comment by agitter** [comment]
> Perhaps we could just try removing it from the dictionary and seeing if it gets picked up by spell check.

I think this is the right solution.  The AppVeyor build above doesn't report "IFN-𝛽-1a" as a spelling error even after removing it from the dictionary.

Based on these discussions, I believe the hyphen is the issue and that it might be non-trivial to modify how aspell treats special characters:
- https://stackoverflow.com/questions/1676132/how-to-use-special-chars-on-aspell-custom-dictionary
- http://www.webspeaks.in/2015/01/adding-special-characters-in-aspell-dictionary-for-php.html

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -291,24 +290,24 @@ Additionally, IL-6 levels remained higher throughout the course of hospitalizati
...
 Synairgen has been investigating a candidate drug, SNG001, which is an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation [@clinicaltrials:NCT04385095] that they reported reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s [@synairgen-SNG001-July-2020; @doi:10/ghjzm4].
-However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-𝛽1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184], although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (Appendix A).
+However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-𝛽-1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184], although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (Appendix A).
```
Comment (rando2) [inline content/20.pharmaceuticals.md:303]:
```suggestion
However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-&beta;-1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184], although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (Appendix A).
```
Thanks for catching that fix! As far as I know, this encoding is the best way to get the Greek letters to render in the docx export!

**Review by rando2 — APPROVED** [review]

**Pull request merged**